sass-importers/glob.js

```diff
@@ -0,0 +1,27 @@
+const glob = require('tiny-glob');
+const path = require('path');
+
+/** @typedef {{file: string} | {contents: string} | null } ResolveResult */
+
+/**
+ * Custom glob importer
+ * Needed to implement this because none of the sass glob importers on npm output @use
+ * @param {string} url Raw path in source code
+ * @param {string} currentPath Path to the current file
+ * @param {(result: ResolveResult) => void} done Callback function for async resolvers
+ * @returns {ResolveResult | void}
+ */
+const globImporter = (url, currentPath, done) => {
+  // This is a really lazy check but it is easy and good enough
+  const isGlob = url.includes('*');
+  if (!isGlob) return null;
+  glob(url, { cwd: path.dirname(currentPath) }).then((matchingPaths) => {
```

Comment:
I could be totally off, but does there need to be a `return` before the `glob` function?

Reply:
I think I am totally off, missing the call to `done`